rotkehlchen/tests/api/test_users.py

```diff
@@ -669,7 +707,8 @@ def test_user_login(rotkehlchen_api_server, username, db_password, data_dir):
         check_proper_unlock_result(result)
 
     assert rotki.user_is_logged_in is True
-    users_data = check_user_status(rotkehlchen_api_server)
+    users_data = check_user_status(rotkehlchen_api_server)  # type: ignore  # TODO: mypy for some reason says that this statement is unreachable. Not sure why.
```

Comment:
you would have to find out. Probably cause it thinks `assert rotki.user_is_logged_in is True` is always False for some reason?